frontend/src/components/applyForm/ApplyForm.tsx

```diff
@@ -42,14 +42,22 @@ const ApplyForm = ({
     formState;
 
   const formObject = !isEmpty(formData) ? formData : savedFormData;
-  const fields = buildForTreeRecursive({
-    errors: validationErrors,
-    formData: formObject,
-    schema: formSchema,
-    uiSchema,
-  });
-
   const navFields = useMemo(() => getFieldsForNav(uiSchema), [uiSchema]);
+  let fields: JSX.Element[] = [];
+  try {
+    fields = buildForTreeRecursive({
```

Comment:
recognize this is an annoying ask at this point where it's already used a bunch, but can we fix the typo now? Every time I look at it I get confused 😅 `buildFormTreeRecursive`

Reply:
Updated, thanks for re-spotting.